config/modules.config

```diff
@@ -36,7 +36,28 @@ process {
         ]
     }
 
-    withName: ALIGNMENT_BAM {
+    withName: FEATURED_ALIGNMENT {
+        ext.alignment_args = [
+            "-q", "20",
+            "-Sb",
+        ].join(' ').trim()
+
+        ext.concoct_cut_up_fasta_args = [
+            "-c 10000",
+            "--merge_last",
+            "-o 0"
+        ].join(' ').trim()
```

Comment:
just curious, do you need the `.trim()` here?

Reply:
I think `.trim()` is useful is you write arguments in list and leave the last argument with comma like [arg1, arg2, arg3, ] (very django :D). Then when you join with (' ') it will give you `'arg1 arg2 arg3 '`. Trim will do `'arg1 arg2 arg3'`. Just saves you from unuseful spaces.
But you are right, I do not need it here :)